### DISCUSSION TITLE
Many Many battles... and i'm losing...

### DISCUSSION THREAD

**Discussion by Owl182-1** [Lolliedieb/lolMiner-releases#681] [Q&A]
Hi,

Apologies for adding this here, i did not know where else to contact you, i am sure you must get hundreds of requests and queries, i have tried for weeks to just get my GPU to run, even for a few minutes, every time i feel like i am getting close there is some other problem, i am sure (i think) that the drivers and everything else are now good but i still get the following error and i just have no idea what to do next...

Setup Miner... 
GPU Health check failed for program build.
GPU Health check failed for create kernel build.
GPU Health check failed for enqueue kernel.
OpenCL driver detected.
Number of OpenCL supported GPUs: 1 
No Cuda driver or GPUs detected. 
Device 0: 
    Name:    Radeon RX 5500 XT (NAVI14, DRM 3.38.0, 5.8.0-59-generic, LLVM 12.0.1) 
    Address: 4294967295:4294967295 
    Vendor:  Intel(R) Corporation
    Drivers: OpenCL
    Memory:  4096 MByte 
    Active:  false (not selected) **_(What does this mean, i am suspicious of this comment)_**

Connecting to pool... 
Connected to eu-eth.ss.btc.com:25  (TLS disabled)
Set Ethash stratum mode: Ethereum Proxy
Authorized worker: 0xacC2eA486f05D5f43Dccd752E0e54609Bd70B121.NightOwlRig1.001 
New job received: 0xa113bb Epoch: 428 Target: 000000007fffffff 
Start Mining... 
All devices deselected or failed compatibility check. Closing lolMinerowl182@owl182-BTC-T37:~/Downloads/Lolminer 1.31$ 

Thanks in advance...

**Comment by elgaarddk** [comment]
Did you solve this? Looking at the same issue with 0 Active.

**Comment by Owl182-1** [comment]
Hi,

Ultimately yes, but are your cards compatible? I found that was my problem but when I moved to RX580 8Gb cards and did a fresh install of Ubuntu my stuff has been running perfectly since

Hope this helps

Wayne

**Reply by elgaarddk** [reply]
Not sure. Trying to make this work on an Intel UHD graphics 620. Is there a compat-list?

**Reply by elgaarddk** [reply]
Oh. It's not supported. :D Nvm. :)

**Comment by Owl182-1** [comment]
It’s all good 😊
